Package.swift

```diff
@@ -55,7 +54,7 @@ let package = Package(
             "SwiftPrometheus",
             "OhhAuth",
             "SwiftSoup",
-            .product(name: "PackageCollectionsModel", package: "SwiftPM")
+            .product(name: "SwiftPMPackageCollections", package: "SwiftPM")
```

Comment:
Unfortunately, this bring with it a warning. I'll try to get rid of it.

<img width="263" alt="CleanShot 2021-11-08 at 07 22 35@2x" src="https://user-images.githubusercontent.com/65520/140693801-7266bbfa-01ab-44e2-8220-0ef51b479b1d.png">